packages/common/project.json

```diff
@@ -28,6 +28,19 @@
         "jestConfig": "packages/common/jest.config.js",
         "passWithNoTests": true
       }
+    },
+    "version": {
+      "executor": "@jscutlery/semver:version",
```

Comment:
@IamBlueSlime Do you have experience working with these plugin before as part of your day job? Or are these being included based just on the discussion we were having in #143

Reply:
Nope, I don't have any on these plugins, I just saw your discussion in the issue while doing this PR, so I've added them. But by reading their README's it seemed pretty accurate to our needs here :/

Reply:
@IamBlueSlime Could you potentially give us some pro/cons of using this rather than other available plugins or ways you would achieve this? I haven used this plugin before but I'm kinda curious

Reply:
@underfisk I never used Nx to publish packages, I'm afraid I can't give you any pro/cons about any package concerning publishing :/